app/src/main/java/org/gdg/frisbee/android/app/OrganizerChecker.java

```diff
@@ -1,85 +1,90 @@
 package org.gdg.frisbee.android.app;
 
 import android.content.SharedPreferences;
+import android.text.format.DateUtils;
 
 import com.google.android.gms.common.api.GoogleApiClient;
 
-import org.gdg.frisbee.android.Const;
 import org.gdg.frisbee.android.api.Callback;
 import org.gdg.frisbee.android.api.model.OrganizerCheckResponse;
 import org.gdg.frisbee.android.utils.PlusUtils;
 import org.gdg.frisbee.android.utils.PrefUtils;
 
 public class OrganizerChecker {
-    private boolean mIsOrganizer = false;
-    private long mLastOrganizerCheck = 0;
-    private String mCheckedId = null;
-    private SharedPreferences mPreferences;
 
-    public OrganizerChecker(SharedPreferences preferences) {
-        mPreferences = preferences;
-        initOrganizer();
-    }
+    private static final String PREF_ORGANIZER_CHECK_TIME = "pref_organizer_check_time";
```

Comment:
I'm thinking about this. Moving preferences related parts to PrefUtils is too much work right now. I don't know if it makes sense. 

One solution to this would be to use its own pref file, separate from the one used in `PrefUtils` That was there will not a any clash. 

Considering this was not in `PrefUtils` I think right now, it is much better.